TombLib/TombLib/Catalogs/TEN Node Catalogs/Lara.lua

```diff
@@ -163,7 +159,30 @@ end
 -- !Description "Checks if Lara is currently mounting a vehicle."
 
 LevelFuncs.Engine.Node.TestLaraVehicle = function()
-    return TEN.Objects.Lara:GetVehicle() ~= nil
+	return TEN.Objects.Lara:GetVehicle() ~= nil
+end
+
+-- !Name "if Lara is on current vehicle..."
+-- !Section "Lara state"
+-- !Conditional "True"
+-- !Arguments "Enumeration, [ Snowmobile | Quad | Speedboat | Kayak | Upv | Minecart | Jeep | Motorbike | Rubber Boat ], 26, type of vehicle"
+-- !Description "Checks if Lara is currently mounting a vehicle and which vehicle it is."
+
+LevelFuncs.Engine.Node.TestLaraVehicleAndType = function(type)
+	local vehicleName = (type == 0) and 'SNOWMOBILE'
```

Comment:
You need to use `Objects.ObjID` constants instead of hardcoded strings here. Otherwise, if object names will be changed under the hood, this node function will break.